nix/tahoe-lafs.nix

```diff
@@ -95,7 +95,7 @@ EOF
   propagatedBuildInputs = with python.pkgs; [
     twisted foolscap zfec appdirs
     setuptoolsTrial pyasn1 zope_interface
-    service-identity pyyaml magic-wormhole treq
```

Comment:
is this why nixos is failing? (we still use `treq` right?)

Reply:
No, it's missing different things in 19.whatever and 20.whatever. Maybe we can drop the former...